README.md

```diff
@@ -89,6 +98,47 @@ kubectl run pwru \
     -- --filter-dst-ip=1.1.1.1 --output-tuple
 ```
 
+Note: You may need to create a volume for `/sys/kernel/debug/tracing` and mount it for the`pwru` pod.
```

Comment:
I'm not sure if we need to include the full pod spec in README. Is it possible to extend the above command @L92 with volume mounts?